tests/discovery.c

```diff
@@ -18,7 +18,8 @@ static void report_sync_error(IotclSyncResponse *response, const char* sync_resp
         printf("IOTC_SyncResponse is NULL. Out of memory?\n");
         return;
     }
-    switch(response->ds) {
+    // FIXME TODO CHECK -- field is renamed?
```

Comment:
I don't think that Neerav has updated tests at all and he should. You probably don't want to be writing or reviewing this this test code until it is ready. Neerav should know how to properly unit-test the code. There should be more to these tests than just fixing compile errors and matching some variable name changes. There's the hello message etc.